ws-nextjs-app/README.md

```diff
@@ -1,11 +1,32 @@
-# WS NextJS App
+# Next.js App
 
-This is an early POC of a NextJS app to test rendering the new Live pages.
+This is the directory that houses the Next.js version of Simorgh.
 
-It utilises existing Simorgh components so as to practice 'evolutionary architecture', we gradually move towards to NextJS app without rebuilding all our React components from scratch. We will gradually migrate across our other page types to this app in the future.
+The Next.js app shares components from the Express app (found in the root of the project). It is intended to be used for new page types, and in the future, will be used for the entire site.
+
+For more information on Next.js and how to use the framework, see the [Next.js documentation](https://nextjs.org/docs).
+
+**Note:** We currently use the 'Pages Router' in Next.js, so be sure to select the "Using Pages Router" tab when viewing the Next.js documentation.
+
+---
 
 ## Setup
 
 - Run `yarn install` to install dependencies
 - Run `yarn dev` to start the development server on `http://localhost:7081`
 - Navigate to `http://localhost:7081/kyrgyz/live/cz74kjpyk07t` to see the page
+
+## Running Unit and E2E tests
+
+- Run `yarn test` to run all unit tests
+- Run `yarn test:e2e` to run all e2e tests
+
+## Running Integration tests
+
+- Navigate up to the root of the project (up a level from here)
+- Run `yarn test:integration --nextJS` to run all integration tests for the Next.js app
```

Comment:
Would it help to include the command for updating the snapshots for the NextJS app?

Reply:
Shall add!